VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.scss

```diff
@@ -9,6 +9,10 @@
   color: govuk-colour('yellow');
 }
 
+.vh-spotlight-self-view-outline {
+  border: 3px solid govuk-colour('yellow');
```

Comment:
This should be fine as px since borders are often an exception, but something to keep an eye out for with the smaller screen support coming up.

Reply:
That is a good point. Will look into this.